README.md

```diff
@@ -102,7 +102,7 @@ resource_tags:
       - "Microsoft.Compute/virtualMachines"
     metrics:
       - name: "CPU Credits Consumed"
-
+    timespan: 10
```

Comment:
The clouwdwatch exporter call it `delay_seconds`, I think it could be a good name also.

Reply:
Ok to call it delay_minutes ?

Reply:
Yes, even though it raises the question to know if we should support "second" granularity. I am not sure about it. But minute should be enough.